app/controllers/pet_schedules_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PetSchedulesController < ApplicationController
-  # skip_before_action :authenticate_user!, only: %i[new create]
+  skip_before_action :authenticate_user!, only: %i[new create]
```

Comment:
in the final version of the project we will have to delete all these skips for authenticate, user's have to be logged in to create a pet

Reply:
I see!! Make sense！

Reply:
We still have time or that, but let's keep it in our minds ;)